marketmaker.js

```diff
@@ -22,6 +25,13 @@ let FEE_TOKEN = null;
 let uniswap_error_counter = 0;
 let chainlink_error_counter = 0;
 
+// Encrypt the eth private key and cryptowatch api key if you want
+let args = minimist(process.argv.slice(2), {boolean: 'encrypt-key'});
```

Comment:
This should use the config.json as well. I would sugest adding a entry like: "encryptKey": true 

Then do the if check like: if(MM_CONFIG?.encryptKey)

Dont think it is a good idea to change the usage for this function. We controlle verything else using the config as well.